cmd/test.go

```diff
@@ -250,7 +249,27 @@ func (t *Test) GetCmd() *cobra.Command {
 
 			passThroughHosts := []models.Filters{}
 
-			err = t.getTestConfig(&path, &proxyPort, &appCmd, &tests, &appContainer, &networkName, &delay, &buildDelay, &ports, &apiTimeout, &globalNoise, &testsetNoise, &coverageReportPath, &withCoverage, configPath, &ignoreOrdering, &passThroughHosts)
+			err = t.getTestConfig(
+				models.TestConfigOptions{
+					Path:               &path,
+					ProxyPort:          &proxyPort,
+					AppCmd:             &appCmd,
+					Tests:              &tests,
+					AppContainer:       &appContainer,
```

Comment:
again instead of defining twice better to assign to a variable and pass it everywhere.

Reply:
We are using TestConfigOptions only once here

Reply:
I checked the `getTestConfig` function compared it to `StartTest` and only one of field is additional : `configPath`.

So a superset called testConfig can be made inside ~
`
testConfig {
testOption
configpath
}
`